### ISSUE TITLE
TypeError: to_bytes() missing required argument 'length' (pos 1)

### ISSUE THREAD

**Issue by BoldMonday** [LucasFonts/vfbLib#41]
Running the latest `vfb3ufo` (Build 2023-10-06) on a vfb file gives the following traceback.
The file in question has been created with FLS 5.1.6 (macOS) and contains name table entries using non-ascii glyphs both in (3, 1, 1033) as well as (1, 0, 0) format.

```
$ vfb3ufo ./"IBM Plex Temp-X03-38-PL.vfb"
VFB3UFO Converter
Copyright (c) 2023 by LucasFonts
Build 2023-10-06
Reading file IBM Plex Temp-X03-38-PL.vfb ...
Source file was successfully read in 94 ms.
Parse error for data: Name Records; ad8b8c8b8bb8cef703f704f70df706f4f2f3f708abbdbbbdbbabd4cdd8abcef703f706f704b9abccf700f700abf706f4f2f3f708f707abf706f0f707f0f706f70af0efb98b8e8cfa9db8cef703f704f70df706f4f2f3f708abbdbbbdbbabd4cdd8abcef703f706f704b9abccf700f700abf706f4f2f3f708f707abf706f0f707f0f706f70af0efb98c8c8b8ba4d4cdd8abdbf700f0f70cabd8ecf708f3abccf700f704f3ecbdbebcbcbdc28c8e8cfa9da4d4cdd8abdbf700f0f70cabd8ecf708f3abccf700f704f3ecbdbebcbcbdc28d8c8b8b92ddf0f2f709f700ecf7068d8e8cfa9d92ddf0f2f709f700ecf7068e8c8b8bb1d4cdd8c6d4cdd8dbf700f0f70cd8ecf708f3b8ccf700f704f3ecbdbebcbcbdc2c6bcb9bbbbbbc6bdbbbdbe8e8e8cfa9db1d4cdd8c6d4cdd8dbf700f0f70cd8ecf708f3b8ccf700f704f3ecbdbebcbcbdc2c6bcb9bbbbbbc6bdbbbdbe8f8c8b8ba4d4cdd8abdbf700f0f70cabd8ecf708f3abccf700f704f3ecbdbebcbcbdc28f8e8cfa9da4d4cdd8abdbf700f0f70cabd8ecf708f3abccf700f704f3ecbdbebcbcbdc2908c8b8b9de1f0f706f707f4f703f702abbcb9bbbbbbabbdbbbdbe908e8cfa9d9de1f0f706f707f4f703f702abbcb9bbbbbbabbdbbbdbe918c8b8ba2d4cdd8dbf700f0f70cd8ecf708f3b8ccf700f704f3ecbdbebcbcbdc2918e8cfa9da2d4cdd8dbf700f0f70cd8ecf708f3b8ccf700f704f3ecbdbebcbcbdc2928c8b8bdcd4cdd8abdbf700f0f70cf73cabf4f707abecabf708f706eceff0f701ecf706f6abf703f1abd4cdd8abcef703f706f704b7abf706f0f2f4f707f708f0f706f0efabf4f702abf701ecf702f70dabf5f709f706f4f707eff4eef708f4f703f702f707abf70bf703f706f700eff70bf4eff0b9928e8cfa9ddcd4cdd8abdbf700f0f70cf742abf4f707abecabf708f706eceff0f701ecf706f6abf703f1abd4cdd8abcef703f706f704b7abf706f0f2f4f707f708f0f706f0efabf4f702abf701ecf702f70dabf5f709f706f4f707eff4eef708f4f703f702f707abf70bf703f706f700eff70bf4eff0b9938c8b8b96cdf703f700efabd8f703f702efecf70d938e8cfa9d96cdf703f700efabd8f703f702efecf70d948c8b8bf4d8f4f6f0abccededf4f702f6b7abdbecf709f700abf70aecf702abeff0f706abd7ececf702b7abdbf4f0f708f0f706abf70aecf702abddf703f707f701ecf700f0f702b7abccf700f0f6f707ecf702eff706ecabdeecf701f709f700f0f702f6f703f70aecb7abd0eff2ecf706abe2ecf700f708f3f0f706f708b7abccf702f702ecabd6f3f703f706ecf707f3948e8cfa9df4d8f4f6f0abccededf4f702f6b7abdbecf709f700abf70aecf702abeff0f706abd7ececf702b7abdbf4f0f708f0f706abf70aecf702abddf703f707f701ecf700f0f702b7abccf700f0f6f707ecf702eff706ecabdeecf701f709f700f0f702f6f703f70aecb7abd0eff2ecf706abe2ecf700f708f3f0f706f708b7abccf702f702ecabd6f3f703f706ecf707f3968c8b8ba4f3f708f708f704c5babaf70bf70bf70bb9edf703f700eff701f703f702efecf70db9eef703f701968e8cfa9da4f3f708f708f704c5babaf70bf70bf70bb9edf703f700eff701f703f702efecf70db9eef703f701978c8b8b9df3f708f708f704c5babaf70bf70bf70bb9f4edf701b9eef703f701978e8cfa9d9df3f708f708f704c5babaf70bf70bf70bb9f4edf701b9eef703f701988c8b8bf724dff3f4f707abd1f703f702f708abdef703f1f708f70becf706f0abf4f707abf700f4eef0f702f707f0efabf709f702eff0f706abf708f3f0abded4d7abdaf704f0f702abd1f703f702f708abd7f4eef0f702f707f0b7abe1f0f706f707f4f703f702abbcb9bcb9abdff3f4f707abf700f4eef0f702f707f0abf4f707abecf70aecf4f700ecedf700f0abf70bf4f708f3abecabd1ccdcabecf708c5abf3f708f708f704c5babaf707eef706f4f704f708f707b9f707f4f700b9f703f706f2badad1d7988e8cfa9df724dff3f4f707abd1f703f702f708abdef703f1f708f70becf706f0abf4f707abf700f4eef0f702f707f0efabf709f702eff0f706abf708f3f0abded4d7abdaf704f0f702abd1f703f702f708abd7f4eef0f702f707f0b7abe1f0f706f707f4f703f702abbcb9bcb9abdff3f4f707abf700f4eef0f702f707f0abf4f707abecf70aecf4f700ecedf700f0abf70bf4f708f3abecabd1ccdcabecf708c5abf3f708f708f704c5babaf707eef706f4f704f708f707b9f707f4f700b9f703f706f2badad1d7998c8b8ba5f3f708f708f704c5babaf707eef706f4f704f708f707b9f707f4f700b9f703f706f2badad1d7998e8cfa9da5f3f708f708f704c5babaf707eef706f4f704f708f707b9f707f4f700b9f703f706f2badad1d79b8c8b8b98d4cdd8abdbf700f0f70cabd8ecf708f39b8e8cfa9d98d4cdd8abdbf700f0f70cabd8ecf708f39c8c8b8b96ccf700f704f3ecbdbebcbcbdc29c8e8cfa9d96ccf700f704f3ecbdbebcbcbdc29e8c8b8bc5d3f703f70babf706ecf70ef703f706edeceef6b8f5f709f701f704f4f702f2abf1f706f703f2f707abeeecf702abf700f0f70af0f700abf707f4f70cabf704f4f705f709f0efabf2f70df701f702ecf707f708f707ac9e8e8cfa9dc5d3f703f70babf706ecf70ef703f706edeceef6b8f5f709f701f704f4f702f2abf1f706f703f2f707abeeecf702abf700f0f70af0f700abf707f4f70cabf704f4f705f709f0efabf2f70df701f702ecf707f708f707ac
Parser class: NameRecordsParser
Traceback (most recent call last):
  File "/Users/paulus/code/vfblib/bin/vfb3ufo", line 8, in <module>
    sys.exit(vfb2ufo())
  File "/Users/paulus/code/vfblib/lib/python3.10/site-packages/vfbLib/cmdline.py", line 187, in vfb2ufo
    vfb.decompile()
  File "/Users/paulus/code/vfblib/lib/python3.10/site-packages/vfbLib/vfb/vfb.py", line 117, in decompile
    entry.decompile()
  File "/Users/paulus/code/vfblib/lib/python3.10/site-packages/vfbLib/vfb/entry.py", line 212, in decompile
    self.decompiled = self.parser().parse(
  File "/Users/paulus/code/vfblib/lib/python3.10/site-packages/vfbLib/parsers/base.py", line 39, in parse
    return self._parse()
  File "/Users/paulus/code/vfblib/lib/python3.10/site-packages/vfbLib/parsers/text.py", line 30, in _parse
    char = c.to_bytes().decode("macroman")
TypeError: to_bytes() missing required argument 'length' (pos 1)
```

**Comment by jenskutilek** [comment]
Ah, the default arguments were only added in Python 3.11, I assume you are using an older version? Fixed in [v0.5.5](https://github.com/LucasFonts/vfbLib/releases/tag/v0.5.5).

**Comment by BoldMonday** [comment]
Yes, running Python 3.10 here.

Updated version seems do to the trick. Thx for the quick fix!
